### PR TITLE
Feature/106

### PR DESCRIPTION
**Pull request by chiragSuccessive**
update order functionality 
[issue-106](https://github.com/nodexpertsdev/RnD/issues/106)

### CODE REVIEW

File: expressEngine/model/collection/order.js [hunk 1]
```diff
@@ -20,6 +20,10 @@ const ordersSchema = new Schema({
     type: String,
     required: false,
   },
+  status: {
+    type: String,
+    required: true,
```
Comment (gkgbunny) [inline expressEngine/model/collection/order.js]: Is required value need to be true in status field?? I think it should be false. If not, please clarify me.

Comment (chiragSuccessive) [reply expressEngine/model/collection/order.js]: @gkgbunny every order have status, like order is canceled, pending, processing, etc.So i think it should be required true.

Comment (aman-successive) [reply expressEngine/model/collection/order.js]: @chiragSuccessive Then you must define the status type to Enum and specify the values in it

Comment (chiragSuccessive) [reply expressEngine/model/collection/order.js]: @aman-successive Deepak is working on it.

File: expressEngine/order/service/order.js [hunk 2]
```diff
@@ -23,6 +24,23 @@ class Service {
...
+      return { error: error.emptyData };
+    }
+    const order = await DBService.updateOne(Order, { orderNumber }, dataToUpdate);
+    if (order.error) {
+      return order;
+    }
+    const { nModified, n } = order;
+    if (nModified) {
+      return { message: success.orderUpdated };
+    } if (n) {
```
Comment (mayank-nx) [inline expressEngine/order/service/order.js]: @chiragSuccessive Fix indentation

File: expressEngine/lib/service/db.js [hunk 3]
```diff
@@ -58,6 +58,14 @@ class DBOperation {
       return { error: err.message };
     }
   }
+
+  updateOne = async (collection, filter, dataToUpdate) => {
+    try {      
+      return await collection.updateOne(filter, dataToUpdate);
```
Comment (mayank-nx) [inline expressEngine/lib/service/db.js]: @chiragSuccessive Please prefer $set

Comment (chiragSuccessive) [reply expressEngine/lib/service/db.js]: ok  sir

**Pull request merged**